src/components/popover-surface/popover-surface.scss

```diff
@@ -52,4 +52,14 @@
             opacity: 1;
         }
     }
+
+    & > * {
+        // this fixes some unwanted layout issues
+        // which might be very hard for consumers to realize
+        // and fix by themselves. Without this code,
+        // if the consumer sets for example paddings in
+        // what they send to the popover, their element
+        // can grow larger than the popover surface
+        box-sizing: border-box;
```

Comment:
<img width="490" alt="image" src="https://github.com/Lundalogik/lime-elements/assets/35954987/b8a66bdb-fc4e-4d23-a8db-7f44c1b67831">
In this example, the component inside the popover doesn't have the `box-sizing: border-box` and a part of it is being pushed out, due to the paddings.

Reply:
How did you reproduce this?